nixos/modules/services/backup/tarsnap.nix

```
@@ -318,7 +325,7 @@ in
         '';
 
         script = let
-          tarsnap = ''tarsnap --configfile "/etc/tarsnap/${name}.conf"'';
+          tarsnap = ''${gcfg.package}/bin/tarsnap --configfile "/etc/tarsnap/${name}.conf"'';
```

```suggestion
          tarsnap = ''${lib.getExe gcfg.package} --configfile "/etc/tarsnap/${name}.conf"'';
```
we could utilize meta.mainProgram here

---

also added meta.mainProgram to the tarsnap package